Gemfile

```diff
@@ -21,6 +21,15 @@ group :test do
   else
     gem 'activesupport', '~> 4.x', require: false
   end
+
+  if Hashie::Extensions::RubyVersion.new(RUBY_VERSION) >= Hashie::Extensions::RubyVersion.new('2.3.0')
+    # gem 'faraday', '~> 0.16.0', require: false
+    gem 'faraday',
+        git: 'https://github.com/BobbyMcWho/faraday.git',
+        branch: 'fix-subclassing-deprecated-classes'
+  else
+    gem 'faraday', '~> 0.15.0', require: false
+  end
```

Comment:
What's the story here? Can we just lock at 0.15 for testing?

Reply:
Sorry, danger was broken on Faraday 0.16.0 and 0.16.1 because of an issue with octokit's use of Faraday errors. I was working on a fix for it last night and wanted to make sure that it fixed the issue I was seeing in Hashie CI. I can close this now that it's done its job in validating